docs/Makefile

```diff
@@ -18,3 +20,21 @@ help:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+# Manually added commands
+# ----------------------------------------------------------------------------
+
+# For local development:
+# - builds and rebuilds html on changes to source
+# - starts a livereload enabled webserver and opens up a browser
+# 
+devenv:
+	sphinx-autobuild -b html --open-browser "$(SOURCEDIR)" "$(BUILDDIR)/html" $(SPHINXOPTS)
+
+# For local development and CI:
+# - verifies that links are valid
+linkcheck:
+	$(SPHINXBUILD) -b linkcheck "$(SOURCEDIR)" "$(BUILDDIR)/linkcheck" $(SPHINXOPTS)
+	@echo
+	@echo "Link check complete; look for any errors in the above output " \
+	      "or in $(BUILDDIR)/linkcheck/output.txt."
```

Comment:
Without this, if you call `make linkcheck`, it still works, because it matches the first generic target:
```
@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS)
```

And it appears to be doing the same thing, i.e. outputs in `$(BUILDDIR)/linkcheck/output.txt`. Also, the `jupyterhub` docs Makefile don't have it:

https://github.com/jupyterhub/jupyterhub/blob/161cdcd7e711845d2c4c157c150f251261a5f925/docs/Makefile#L20-L29

Do we wish to have it here for the extra info echo message? What do you think?

Reply:
Sorry I ended up merging before reading this comment, nice catch! Opened #23 about this